cpe/openscap-cpe-dict.xml

```diff
@@ -142,4 +142,12 @@
                  We assume that the productivity is equal to RHEL5. We are unable to write a better OVAL check. -->
             <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="openscap-cpe-oval.xml">oval:org.open-scap.cpe.rhel:def:5</check>
       </cpe-item>
+      <cpe-item name="cpe:/o:windriver:wrlinux">
+            <title xml:lang="en-us">Wind River Linux all versions</title>
+            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="openscap-cpe-oval.xml">oval:org.open-scap.cpe.wrlinux:def:1</check>
```

Comment:
Here you reference `oval:org.open-scap.cpe.wrlinux:def:1`, but there is no definition with that ID in `cpe/openscap-cpe-oval.xml`. Every check referenced from the cpe_items must be defined in `cpe/openscap-cpe-oval.xml`